types/openid-client/index.d.ts

```diff
@@ -167,13 +172,21 @@ export class Client {
      * @param extras Additional revoke options
      */
     revoke(token: string, tokenTypeHint: string, extras?: RevokeRequestOptions): Promise<void>;
+
+    /**
+     * Refresh your active token
+     * @param refreshToken The refresh token
+     * @param opts Additional options
+     */
+    refresh(refreshToken: string, opts?: RefreshRequestOptions): Promise<TokenSet>;
 }
 
 export class TokenSet {
     readonly access_token?: string;
     readonly token_type?: string;
     readonly id_token?: string;
     readonly refresh_token?: string;
+    readonly expiration?: number;
```

Comment:
`expiration` is not mentioned here: https://github.com/panva/node-openid-client/blob/master/docs/README.md#new-tokensetinput

Do you mean `expires_at`?

Maybe it's a custom value from your OIDC implementation, then it still shouldn't be part of this generic interface.